packages/react-router-dom/__tests__/link-href-test.tsx

```diff
@@ -193,7 +193,7 @@ describe("<Link> href", () => {
         );
       });
 
-      expect(renderer.root.findByType("a").props.href).toEqual("/inbox");
+      expect(renderer.root.findByType("a").props.href).toEqual("/");
```

Comment:
This `..` should ignore the `index` route and thus be relative to `inbox` which would resolve to the root.  The incorrect nature of this test can be seen by comparing to the test above where in this same layout, `<Link to=".">` resolves to `/inbox` - so `..` should resolve to `/`